src/api/connector/external/rest.js

```diff
@@ -1,16 +1,102 @@
-/*
-Generic HTTP REST connector.
-This connector should implement OPEN API specification.
-https://github.com/OAI/OpenAPI-Specification
-https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md
-*/
+import http from 'axios';
+import { toLower } from 'lodash';
+import { logger } from '../../../utility';
+
+const getIdentifier = (source, options) => {
+  const identifierName = source.schema.identifier;
+  const identifier = options.params[identifierName];
+
+  if (!identifier) {
+    logger.error('Identifier field not found in params');
+    throw new Error('Identifier field not found in params');
+  }
+
+  return identifier;
+};
+
+const getSortPrefix = (sortBy) => {
+  const sortByParam = toLower(sortBy);
+  let sortPrefix = '';
+
+  if (sortByParam === 'desc') sortPrefix = '-';
+  else if (sortByParam === 'asc') sortPrefix = '+';
+
+  return sortPrefix;
+};
+
+const getApiParams = (clientParams) => {
+  const apiParams = {};
+
+  // TODO: Add field filter handling
+  apiParams.sort = `${getSortPrefix(clientParams.sortBy)}${clientParams.sort}`;
```

Comment:
So, client params would be:
```
{
   sort: 'age',
   sortBy: 'asc'
}
```

Wouldn't it be more straightforward to change these two prop names so `sortBy` has the value of sorting param, and `sort` has sorting direction? 🤔

Reply:
Yeah, more _straightforward_ could be used to describe it, but I'd say that more _correct_ is even more appropriate :)

Reply:
Me at my subtlest 😄